storage/allocator_test.go

```diff
@@ -46,141 +53,135 @@ var multiDCConfig = proto.ZoneConfig{
 	},
 }
 
-// filterStores returns just the store descriptors in the supplied
-// stores slice which contain all the specified attributes.
-func filterStores(a proto.Attributes, stores []*proto.StoreDescriptor) ([]*proto.StoreDescriptor, error) {
-	var filtered []*proto.StoreDescriptor
+func gossipStores(g *gossip.Gossip, stores []*proto.StoreDescriptor, t *testing.T) {
+	var wg sync.WaitGroup
```

Comment:
this machinery (wrapping the iteration of `stores`) looks duplicated. can it be extracted?